lib/kitchen/directions/bake_chapter_answer_key/strategies/uphysics.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeChapterAnswerKey
+  module Strategies
+    class UPhysics
+      def bake(chapter:, append_to:)
+        classes = %w[review-conceptual-questions review-problems review-additional-problems
+                     review-challenge]
+        classes.each do |klass|
+          bake_section(chapter: chapter, append_to: append_to, klass: klass)
```

Comment:
is there a reason for the k in klass? maybe with can it selector or something more specific

Reply:
oh it's bc `class` is already a thing in Ruby. I do like selector, except for that it seems to imply we could use an xpath as well. Using `klass` is a thing I borrowed from JP. Maybe `tag`?

Reply:
`tag` makes me think its referring to the html tag. 
I think `klass` is fine here?